odoo/tests/common.py

```diff
@@ -1764,7 +1764,7 @@ def authenticate(self, user, password):
 
         return session
 
-    def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=None, error_checker=None, watch=False, **kw):
+    def browser_js(self, url_path, code, ready='', login=None, timeout=120, cookies=None, error_checker=None, watch=False, **kw):
```

Comment:
Do you have example of tests longer than that? Shouldn't the parameter be given by those tests?